### PR TITLE
docs: fix typos

### PR DESCRIPTION
**Pull request by todoroff**
Caught and fixed some typos :)

### CODE REVIEW

File: doc/async_worker_variants.md [hunk 1]
```diff
@@ -418,7 +418,7 @@ void Napi::AsyncProgressQueueWorker::ExecutionProcess::Send(const T* data, size_
 ## Example
 
 The code below show an example of the `Napi::AsyncProgressQueueWorker` implementation, but
```
[the remark this answers is not in the document]
Comment (todoroff) [reply doc/async_worker_variants.md]: Missed that one. Corrected 👍

**Review by NickNaso — APPROVED** [review]
LGTM

**Review by mhdawson — APPROVED** [review]
LGTM

**Comment by mhdawson** [comment]
@todoroff thanks!

**Pull request merged**